### PR TITLE
13565 UI tweaks per Yui

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#13565

*Description of changes:*
- added some font size CSS variables
- small UI tweaks per Yui
- app version = 5.7.8

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dashboard/Alerts/MissingInformation.vue [hunk 1]
```diff
@@ -54,14 +54,46 @@ export default class MissingInformation extends Vue {
   pointer-events: none; // disable whole-row expansion
 }
 
+h3 {
+  .v-icon {
+    font-size: $px-18;
+  }
+
+  span {
+    font-size: $px-16;
```
Comment (severinbeauvais) [inline src/components/Dashboard/Alerts/MissingInformation.vue:63]: I'm waiting to hear back from Yui on whether this should be 14px instead.

**Review by lambert-alex — APPROVED** [review]
LGTM

**Review by lewischenstudio — APPROVED** [review]
LGTM.

**Comment by severinbeauvais** [comment]
Before and after:

image

File: src/components/Dashboard/Alerts/MissingInformation.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/Dashboard/Alerts/MissingInformation.vue:63]: No answer in 1 hour... merging anyway. (Can change later if important.)

**Pull request merged**